R/get_p_value.R

```diff
@@ -12,7 +12,8 @@
 #' @param obs_stat A numeric value or a 1x1 data frame (as extreme or more
 #'   extreme than this).
 #' @param direction A character string. Options are `"less"`, `"greater"`, or
-#'   `"two_sided"`. Can also use `"left"`, `"right"`, or `"both"`.
+#'   `"two_sided"`. Can also use `"left"`, `"right"`, `"both"`, 
```

Comment:
I would suggest promoting either two-sided or two sided to be listed first and put two_sided as one of the "can also use"

Reply:
Sure thing! I'll get to this once I can get Github Actions builds passing again.